apph-back/src/main/java/com/viseo/apph/service/TagService.java

```diff
@@ -41,15 +39,15 @@ public Set<Tag> createListTags(String listOfTags, User user) {
             throw new IllegalArgumentException("photo.error.tagOverChar");
         Set<Tag> allTags = Arrays.stream(parsedTags).filter(tag -> tag.getId() != 0).collect(Collectors.toSet());
         for (Tag tag : tagsToCreate) {
-            Tag newTag = tagDao.createTag(new Tag().setName(tag.getName().substring(14)).setUser(user));
+            Tag newTag = tagDao.createTag(new Tag().setName(tag.getName().substring(2)).setUser(user));
             allTags.add(newTag);
         }
         return allTags;
     }
 
     Boolean isTagValid(List<Tag> list) {
         for (Tag tag : list) {
-            if (tag.getName().substring(14).length() > 255)
+            if (tag.getName().substring(2).length() > 255)
```

Comment:
```suggestion
            if (tag.getName().length() > 255 + 2)
```

autant pas faire de substring pour rien c'est couteux